pkg/liqoctl/status/local/localinfo_test.go

```diff
@@ -110,82 +91,76 @@ var _ = Describe("LocalInfo", func() {
 			Expect(lic.localInfoSection).To(Equal(output.NewRootSection()))
 		})
 	})
-	Context("Collecting and Formatting LocalInfoChecker", func() {
-		When("Standard case", func() {
-			BeforeEach(func() {
-				clientBuilder.WithObjects(
-					append(
-						baseObjects,
-						forgeLiqoAuth(namespace, ""),
-					)...,
-				)
-				options.CRClient = clientBuilder.Build()
-				lic = NewLocalInfoChecker(&options)
-				lic.Collect(ctx)
-			})
-			JustBeforeEach(func() {
-				text = lic.Format()
-				text = pterm.RemoveColorFromString(text)
-				text = testutil.SqueezeWhitespaces(text)
-			})
-			It("should not return errors", func() {
-				Expect(lic.HasSucceeded()).To(BeTrue())
-			})
-			It("should format a valid text", func() {
-				Expect(text).To(ContainSubstring(
-					pterm.Sprintf("Cluster ID: %s", clusterID),
-				))
-				Expect(text).To(ContainSubstring(
-					pterm.Sprintf("Cluster Name: %s", clusterName),
-				))
-				for _, v := range testutil.ClusterLabels {
-					Expect(text).To(ContainSubstring(v))
-				}
-				Expect(text).To(ContainSubstring(
-					pterm.Sprintf("Pod CIDR: %s", testutil.PodCIDR),
-				))
-				Expect(text).To(ContainSubstring(
-					pterm.Sprintf("Service CIDR: %s", testutil.ServiceCIDR),
-				))
-				Expect(text).To(ContainSubstring(
-					pterm.Sprintf("External CIDR: %s", testutil.ExternalCIDR),
-				))
-				Expect(text).To(ContainSubstring(
-					pterm.Sprintf("Address: %s", fmt.Sprintf("https://%v:6443", testutil.APIAddress)),
-				))
-				for _, v := range testutil.ReservedSubnets {
-					Expect(text).To(ContainSubstring(v))
-				}
 
-			})
-		})
-		When("API server is overrided", func() {
-			BeforeEach(func() {
-				clientBuilder.WithObjects(
-					append(
-						baseObjects,
-						forgeLiqoAuth(namespace, testutil.OverrideAPIAddress),
-					)...,
-				)
-				options.CRClient = clientBuilder.Build()
-				lic = NewLocalInfoChecker(&options)
-				lic.Collect(ctx)
-			})
-			JustBeforeEach(func() {
-				text = lic.Format()
-				text = pterm.RemoveColorFromString(text)
-				text = testutil.SqueezeWhitespaces(text)
-			})
-			It("should format a valid text", func() {
-				Expect(text).To(ContainSubstring(
-					pterm.Sprintf("Address: %s", fmt.Sprintf("https://%v", testutil.OverrideAPIAddress)),
-				))
-			})
-		})
-	})
+	DescribeTable("Collecting and Formatting LocalInfoChecker", func(clusterLabels, apiServerOverride bool) {
```

Comment:
You can encapsulate the test case arguments in a struct, which makes the entries' definitions more readable.